src/game.py

```diff
@@ -47,7 +47,26 @@ def create(self, screen, messages):
             self.image = self.font.render('NO IMAGE', False, (255, 255, 255))
 
         self.label = self.font.render(self.title, False, (255, 255, 255))
-        self.label_desc = self.font.render(self.description, False, (255, 255, 255))
+        self.desc_lines = self.wrapDesc(self.description, self.font)
+    
+    def wrapDesc(self, description, font):
+        self.lines = []
+        self.desc_split = description.split(" ")
+        self.curr_width = font.size(description)[0]
+        self.line2 = ""
```

Comment:
Is there a particular reason why ```self.line2``` makes more sense then, let's say, ```self.new_line```?

Reply:
Not at all! I forgot about that, but it was from before I let it deal with more line breaks. Sorry!

Reply:
Just change this variable name and then this is perfect!